### DISCUSSION TITLE
reverse changes to Air Tux jumping on springs

### DISCUSSION THREAD

**Discussion by Alasdairbugs** [SuperTux/supertux#2455] [Ideas]
I think its kinda weird that the trampolines dont give you extra height with an air flower now, since that adds an extra element of unique-ness to the air flower, and a potential way to deliver a secret.

The changes made in the recent nightlies, where tux just has the same jump height as big tux should be reverted i think. This doesn't really affect story mode, as i think most players wouldn't hold onto the air flower across levels. I think most people just grab powerups as they are given.

**Comment by Alasdairbugs** [comment]
also there was an alternative suggestion by orange:

ground-pounding on springs should give a higher spring jump?

FYI, im using springs synonymously with trampolines.

**Comment by weluvgoatz** [comment]
Air Tux still gives you a certain boost with springs; it's just been significantly decreased because it was an extremely overpowered amount of height you'd get from those springs with Air Tux.  Either you'd basically get to jump out of the entire level or bang your head abruptly if the spring was in an area with a roof.  It can be increased to be more noticeable, but it should not be to the degree that it was previously.
